src/neptune/management/internal/utils.py

```diff
@@ -61,3 +64,21 @@ def to_api(self) -> str:
         if self.value == "admin":
             return "owner"
         return self.value
+
+
+def get_trashed_object_ids(project: Optional[str], api_token: Optional[str]) -> List[str]:
+    from neptune import init_project  # to avoid circular imports
+
+    with init_project(project=project, api_token=api_token) as project_obj:
```

Comment:
Let's not use a high-level `neptune` API here. `management` and `neptune`  packages should be built on top of some common `api` package and be treated as a some kind of siblings. So let's use NQL building here. Especially that:
- this will prevent us from circular dependencies probably
- probably you'll be able to list all objects at once (runs & models), don't remember but I expect that it's possible